src/main/java/core/basesyntax/models/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle implements Figure {
+    private final String color;
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color
+                + "Type: Right Triangle "

```

Comment:
The `println` statement is missing a space before the word 'Type'. Without this space, the output will concatenate the color and the word 'Type', making it less readable.